ansible/roles/dnsmasq/tasks/main.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Install dnsmasq
+  package:
+    name=dnsmasq state=present
+  tags:
+    - setup
+
+- name: Create dnsmasq.d
+  file:
+    path=/etc/dnsmasq.d state=directory owner=root group=root mode=0755
```

Comment:
This is a directory for drop-in configuration files from other services.

http://blog.siphos.be/2013/05/the-linux-d-approach/